docs/ui/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, shadows, spacing, theme } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { HEADLINE } from '~/ui/components/Text';
+import { durations } from '~/ui/foundations/durations';
+import { ChevronDownIcon } from '~/ui/foundations/icons';
+
+type CollapsibleProps = PropsWithChildren<{
+  /** The content of the collapsible summary */
+  summary: ReactNode;
+  /** If the collapsible should be rendered "open" by default */
+  open?: boolean;
+  testID?: string;
+}>;
+
+export function Collapsible({ summary, open, testID, children }: CollapsibleProps) {
+  return (
+    <details css={detailsStyle} open={open} data-testid={testID}>
+      <summary css={summaryStyle}>
+        <ChevronDownIcon css={markerStyle} size={iconSize.small} />
+        <HEADLINE tag="span">{summary}</HEADLINE>
+      </summary>
+      <div css={contentStyle}>{children}</div>
+    </details>
+  );
+}
+
+const detailsStyle = css({
+  overflow: 'hidden',
+  background: theme.background.default,
+  border: `1px solid ${theme.border.default}`,
+  borderRadius: borderRadius.medium,
+  padding: 0,
+
+  '&[open]': {
+    boxShadow: shadows.tiny,
+  },
+});
+
+const summaryStyle = css({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  userSelect: 'none',
+  backgroundColor: theme.background.secondary,
+  padding: spacing[1.5],
+  margin: 0,
+
+  '&::marker': { content: '""' },
+});
+
+const markerStyle = css({
+  margin: `0 ${spacing[1.5]}px`,
+  transform: 'rotate(-90deg)',
+  transition: `transform ${durations.hover}`,
```

Comment:
@jonsamp I think we want to add a new duration for this? Even though, this seems to be working pretty well.

Reply:
We can do a custom one. Maybe 200ms or 250ms.